### PR TITLE
Refactor/file separation

### LINKED ISSUE #18: Create API Calls
Add API call folder, attach to App
Clean Data to reflect PlanetBio interface

### LINKED ISSUE #19: Create Interface File
Create Typescript interface file and attach to  components as needed

### PR DESCRIPTION
**Pull request by ConnorAndersonLarson**
# PR PlanetParty
**Connor A-L, Alex T & Katie B**

### What does this do?

- [ ] Fix
- [ ] Feature
- [X] Refactor
- [ ] Style
- [ ] Testing

### Are there any known bugs?

- [ ] No
- [X] Yes (if so please disclose in Notes for Reviewer & Next Iterations)

### Summary of the changes
Places interfaces into separate file to be called externally from components
Creates API file to hand fetch requests
Cleans GET request data to only use data we need

### What ticket(s) do the changes resolve?
Close #18 
Close #19 

Please Link Issues to this PR as necessary.
#18 
#19 

### Notes for the Reviewer
N/A

### How to Test Changes?
Pull Down changes, open via `npm start`. Site should display with 8 planets (all same image) and unique names. Checking App state in React Dev Tools should show all planet objects inside state.

### Next Iterations
Translate apiCalls.js into ts, utilize interface.

### CODE REVIEW

File: src/App/App.tsx [hunk 1]
```diff
@@ -1,23 +1,11 @@
...
-  length_of_day: number,
-  distance_from_sun: number,
-  mean_temperature: number,
-  number_of_moons: number,
-}
-
-interface AllData {
-  allPlanets: Array<PlanetBio>
-}
+console.log(discoverPlanets())
```
Comment (alexthompson207) [inline src/App/App.tsx:8]: Note for later to remove this

File: src/App/App.tsx [hunk 2]
```diff
@@ -26,6 +14,12 @@ class App extends React.Component<{}, AllData> {
       allPlanets: planetData
     };
   }
+
+  componentDidMount = () => {
+    discoverPlanets()
+      .then(result => this.setState({allPlanets: result}))
+  }
```
Comment (alexthompson207) [inline src/App/App.tsx:21]: Later we can implement a catch here for check for server errors

**Review by alexthompson207 — COMMENTED** [review]
This looks great, good work with cleaning the API data

File: src/App/App.tsx (hunk 2 above)
Comment (knees4bees) [reply src/App/App.tsx:21]: I'll make an issue for it now so we don't forget. It's issue #26

File: src/apiCalls.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+      mass: info.mass,
+      diameter: info.diameter,
+      gravity: info.gravity,
+      length_of_day: info.length_of_day,
+      distance_from_sun: info.distance_from_sun,
+      mean_temperature: info.mean_temperature,
+      number_of_moons: info.number_of_moons
+    }
+  }))
+  .then(response => response.filter(planet => (planet.name !== 'Moon' && planet.name !== 'Pluto')))
```
Comment (alexthompson207) [inline src/apiCalls.js:17]: Great work with this method to filter!!!

**Review by knees4bees — APPROVED** [review]

**Review by alexthompson207 — COMMENTED** [review]
Great work with this, see comment!

**Pull request merged**